package.json

```diff
@@ -1,20 +1,19 @@
 {
   "name": "carbone-docker",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "description": "All what you need to build a standalone carbone.io Docker image",
   "main": "index.js",
   "author": "Florian Bezagu <florian@bezagu.com>",
   "license": "MIT",
   "dependencies": {
-    "body-parser": "^1.18.3",
-    "carbone": "^1.2.0",
-    "express": "^4.16.4",
-    "fs-extra": "^7.0.1",
-    "lodash": "^4.17.15",
-    "multer": "^1.4.1",
-    "telejson": "^3.0.3"
+    "body-parser": "^1.19.0",
+    "carbone": "^2.0.0",
+    "express": "^4.17.1",
+    "fs-extra": "^9.0.1",
```

Comment:
We can remove this package

Reply:
Why ? It's used in `index.js`

Reply:
Sorry didn't see the other comment about the native `promises`

Reply:
Yes and can be replaced with native build in `const fs = require(`fs`).promises;` so we remove extra dependency and remove few kb :smile: